apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -79,6 +79,21 @@ type StorageSizes struct {
 	PerWorkspace *resource.Quantity `json:"perWorkspace,omitempty"`
 }
 
+type ServiceAccountConfig struct {
+	// ServiceAccountName defines a fixed name to be used for all DevWorkspaces. If set, the DevWorkspace
+	// Operator will not generate a separate ServiceAccount for each DevWorkspace, and will instead create
+	// as ServiceAccount with the specified name in each namespace where DevWorkspaces are created. If specified
```

Comment:
Small typo: as -> a
`...will instead create a* ServiceAccount with the specified name in each namespace...`